### PR TITLE
Features/random texts

### PR DESCRIPTION
**Pull request by ConnorAndersonLarson**
Adds the randomization features of Iteration 0 and Iteration 1.
Created event listeners to randomize covers on demand.
main.js has all code creation on it, inside main.js line 26 has the beginning of primary function of this pull, randomCover()
open index.html in browser. Refresh should yield random covers as well as the 'Random Cover' button.
See attached for working feature gif.
Iteration1

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -17,3 +21,20 @@ var currentCover;
 function getRandomIndex(array) {
   return Math.floor(Math.random() * array.length);
 }
+new Cover (randomCover())
```
Comment (itsnameissteven) [inline src/main.js]: I think we can assign this to var currentCover

File: src/main.js [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
+var taglineTwo = document.querySelector('.tagline-2')
+var randomCoverButton = document.querySelector('.random-cover-button')
 
 // We've provided a few variables below
 var savedCovers = [
   new Cover("http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg", "Sunsets and Sorrows", "sunsets", "sorrows")
 ];
-var currentCover;
-
+var currentCover = randomCover();
```
Comment (itsnameissteven) [inline src/main.js]: assign the new Cover here maybe ??????

**Review by itsnameissteven — CHANGES_REQUESTED** [review]
Just one small change maybe? 

randomCover function looks great.

**Comment by ConnorAndersonLarson** [comment]
Added changes, see new PR

**Comment by ConnorAndersonLarson** [comment]
Added changes per request, tested for future features to potentially be compatible.

**Pull request merged**